introduction/boolean.md

```diff
@@ -0,0 +1,21 @@
+## Boolean
+
+Boolean means True or False. Would it surprise you to find out that the fundamentals of computation are built entirely out of True and False? We often call this concept "binary" -- meaning two choices. You'll often here these two names, "binary" and "boolean" used to describe the same concept.
+
+One of the primary things a programmer does everyday is figuring out how to take a human concept and turn it into something that the computer can run. Being such a fundamental building block of programming means that really complex processes can be broken down into chunks of what we can call "boolean logic".
+
+Our REPL uses boolean logic.
+
+The Read phase is boolean: "has the user hit Enter/Return yet"? If the answer is yes/true, then it takes whatever text was input before and sends that input to the Evaluate phase.
```

Comment:
Have we introduced the REPL yet? How to access it? Otherwise we might need more context.